dbms/src/Flash/Mpp/ExchangeReceiver.cpp

```diff
@@ -486,6 +485,7 @@ void ExchangeReceiverBase<RPCContext>::setUpConnection()
             });
 
             ++thread_count;
+            --connection_uncreated_num;
         }
```

Comment:
call `--connection_uncreated_num;` here and don't need to care sync/local/async?

Reply:
> call `--connection_uncreated_num;` here and don't need to care sync/local/async?

yes, it's needless to care about the uncreated connection type.

Reply:
I mean, just call `--connection_uncreated_num;` in L489, no need to call `--connection_uncreated_num;` elsewhere

Reply:
> I mean, just call `--connection_uncreated_num;` in L489, no need to call `--connection_uncreated_num;` elsewhere

okk

Reply:
> I mean, just call `--connection_uncreated_num;` in L489, no need to call `--connection_uncreated_num;` elsewhere

According to @yibin87 advice, we need to do `--connection_uncreated_num;` in `if-else`, not in `for-loop`, because `connection_uncreated_num` should not be reduced when this is an async connection.

Reply:
ok

> > I mean, just call `--connection_uncreated_num;` in L489, no need to call `--connection_uncreated_num;` elsewhere
> 
> According to @yibin87 advice, we need to do `--connection_uncreated_num;` in `if-else`, not in `for-loop`, because `connection_uncreated_num` should not be reduced when this is an async connection.

get